container/image_test.py

```diff
@@ -91,7 +91,7 @@ def test_files_in_layer_with_file_base(self):
 
   def test_tar_base(self):
     with TestImage('tar_base') as img:
-      self.assertDigest(img, '611270a900f1d95f896af3d8c38ff04f75d75f972c02eab0403aa12847881489')
+      self.assertDigest(img, '9fdc5d975efcc73690089a2eedaa28557ce7a792baa1fcd5e56949fb637c38f2')
```

Comment:
What's the diff in the new image that results in the digest mismatch?

Reply:
Per my other comment, the version of TarFileWriter in rules_pkg defaults to preserving input tar file mtimes. I have verified that these digests have the original values if I turn off that functionality, so that seems to be the only change. If you'd prefer, I'm open to splitting this into two PRs: one that introduces rules_pkg as a dependency with the new functionality turned off, and then another that turns the mtime-preserving functionality on.